### ISSUE TITLE
Regression: Minio returns garbage when doing a GET ...?acl request

### ISSUE THREAD

**Issue by fviard** [minio/minio#6008]
<!--- Provide a general summary of the issue in the Title above -->

I have s3cmd automatic test that is running on travis-ci using Minio.
Usually, Minio is downloaded once and cached, so not often updated.

On my personal computer where I have the same issue, the automatic test was working with Minio version of the 22/11/2017.

So today Minio was updated and the test is not passing anymore.
More than that, the test is failing because for one request, minio is returning binary garbage instead of a normal xml reply.

From my debug log, it looks like the binary content comes as a reply to a GET ...file?acl request.

## Expected Behavior
Get ..file?acl returns a xml reply.
The s3cmd test suite pass.

## Current Behavior
Get ...?acl returns unknown binary.

## Context
In the following I will paste 2 logs:
- The log of travis-ci that shows the issue and the previous tests.
- The log on my computer on the request that is failing with debug outputs.


===Log of travis CI ===
https://travis-ci.org/s3tools/s3cmd/jobs/387427865
```
git.checkout
1.53s$ git clone --depth=50 --branch=master https://github.com/s3tools/s3cmd.git s3tools/s3cmd
Cloning into 's3tools/s3cmd'...
remote: Counting objects: 323, done.
remote: Compressing objects: 100% (137/137), done.
remote: Total 323 (delta 199), reused 268 (delta 183), pack-reused 0
Receiving objects: 100% (323/323), 492.55 KiB | 7.58 MiB/s, done.
Resolving deltas: 100% (199/199), done.
$ cd s3tools/s3cmd
$ git checkout -qf 4fb04d57ab0a9820f988340681ba4f0da3b57b10
0.01s$ source ~/virtualenv/python2.7/bin/activate
cache.1
Setting up build cache
$ export CASHER_DIR=$HOME/.casher
0.06s$ Installing caching utilities
0.01s
8.25sattempting to download cache archive
fetching master/cache-linux-trusty-e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855--python-2.7.tgz
fetching master/cache--python-2.7.tgz
could not download cache
0.00s
1.21sadding /home/travis/cache to cache
creating directory /home/travis/cache
$ python --version
Python 2.7.14
$ pip --version
pip 9.0.1 from /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (python 2.7)
before_install.1
0.01s$ mkdir -p $HOME/cache
before_install.2
1.30s$ test ! -e $HOME/cache/minio && wget -O $HOME/cache/minio https://dl.minio.io/server/minio/release/linux-amd64/minio || echo "Minio already in cache"
--2018-06-03 15:22:25--  https://dl.minio.io/server/minio/release/linux-amd64/minio
Resolving dl.minio.io (dl.minio.io)... 162.243.132.171
Connecting to dl.minio.io (dl.minio.io)|162.243.132.171|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 24223904 (23M) [application/octet-stream]
Saving to: ‘/home/travis/cache/minio’
100%[======================================>] 24,223,904  24.4MB/s   in 0.9s   
2018-06-03 15:22:27 (24.4 MB/s) - ‘/home/travis/cache/minio’ saved [24223904/24223904]
before_install.3
0.01s$ mkdir -p $HOME/minio_tmp
install
1.75s$ pip install .
Processing /home/travis/build/s3tools/s3cmd
Collecting python-dateutil (from s3cmd===2.0.1-)
  Downloading https://files.pythonhosted.org/packages/cf/f5/af2b09c957ace60dcfac112b669c45c8c97e32f94aa8b56da4c6d1682825/python_dateutil-2.7.3-py2.py3-none-any.whl (211kB)
    100% |████████████████████████████████| 215kB 4.4MB/s 
Collecting python-magic (from s3cmd===2.0.1-)
  Downloading https://files.pythonhosted.org/packages/42/a1/76d30c79992e3750dac6790ce16f056f870d368ba142f83f75f694d93001/python_magic-0.4.15-py2.py3-none-any.whl
Requirement already satisfied: six>=1.5 in /home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages (from python-dateutil->s3cmd===2.0.1-)
Installing collected packages: python-dateutil, python-magic, s3cmd
  Running setup.py install for s3cmd ... done
Successfully installed python-dateutil-2.7.3 python-magic-0.4.15 s3cmd-2.0.1-
before_script.1
0.00s$ export AWS_ACCESS_KEY_ID=Q3AM3UQ867SPQQA43P2F
before_script.2
0.00s$ export AWS_SECRET_ACCESS_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
before_script.3
0.00s$ export MINIO_ACCESS_KEY=Q3AM3UQ867SPQQA43P2F
before_script.4
0.00s$ export MINIO_SECRET_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
before_script.5
0.01s$ chmod +x $HOME/cache/minio
before_script.6
0.01s$ $HOME/cache/minio server $HOME/minio_tmp &
before_script.7
4.01s$ sleep 4
Created minio configuration file successfully at /home/travis/.minio
Endpoint:  http://172.17.0.12:9000  http://127.0.0.1:9000
AccessKey: Q3AM3UQ867SPQQA43P2F 
SecretKey: zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG 
Browser Access:
   http://172.17.0.12:9000  http://127.0.0.1:9000
Command-line Access: https://docs.minio.io/docs/minio-client-quickstart-guide
   $ mc config host add myminio http://172.17.0.12:9000 Q3AM3UQ867SPQQA43P2F zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG
Object API (Amazon S3 compatible):
   Go:         https://docs.minio.io/docs/golang-client-quickstart-guide
   Java:       https://docs.minio.io/docs/java-client-quickstart-guide
   Python:     https://docs.minio.io/docs/python-client-quickstart-guide
   JavaScript: https://docs.minio.io/docs/javascript-client-quickstart-guide
   .NET:       https://docs.minio.io/docs/dotnet-client-quickstart-guide
10.30s$ python ./run-tests-minio.py -c .travis.s3cfg -p baseauto
System encoding: UTF-8
Using bucket prefix: 'baseauto'
  1  Remove test buckets ..... OK
  2  Verify no test buckets .. OK
  3  Create one bucket ....... OK
  4  Create multiple buckets . OK
  5  Invalid bucket name ..... OK
  6  Buckets list ............ OK
  7  Sync to S3 .............. OK
  8  Sync UTF-8 to S3 ........ OK
  9  List bucket content ..... OK
 10  List bucket recursive ... OK
 11  Recursive put ........... OK
 12  Clean testsuite-out/(rm)  SKIP
 13  Clean testsuite-out/(mk)  OK
 14  Put from stdin .......... OK
 15  Put multipart ........... OK
 16  Multipart large put from stdin  OK
 17  Clean testsuite-out/(rm)  OK
 18  Clean testsuite-out/(mk)  OK
 19  Put multiple files ...... OK
 20  Move without '/' ........ OK
 21  Move recursive w/a '/' .. OK
 22  Move multiple files ..... FAIL  ('retcode: 1, expected one of: [0]')
----
python s3cmd -c .travis.s3cfg mv s3://baseautos3cmd-autotest-1/urandom1.bin s3://baseautos3cmd-autotest-1/urandom2.bin s3://baseautos3cmd-autotest-1/dir/
----
ERROR: Error parsing xml: not well-formed (invalid token): line 1, column 0
ERROR: �B��Yd��������ȝ��[�_��f�Cm{~���v�Py���Eᙥ���J4p/���hs"wY����MZB<�bIv�ϰ��ك���10�As,�U G�G }l6���s�F�̹
�Ԅ
	]�i@���ʸ�OU�=����4�b�2��1;\^Z�G�����xRjO=ֹg�H��u6�!�0���1LRV��j��\(/�L�M�I��#&t^�k�����Qo%D�YL��[8;�hɂ�8�3j���Κ��׼�1n���	H	�-�d�g���c\
�Z��d/�K���ޫ
�^�ۊ{dbzk��������ZJE� �����y���wU��9ԡO���I�D�*���Go��]��k� ���
[����߷�5�#���$AA_��o�8��h�(�v��ʹb#�&�5�|�[�����M��:nO��Ұ����[�+t[�c���qb���lVC:�U�Yw���ƙ�|6�n˛LR��c
|~+�Ih79�
�H�rIx��@�u��}&S�w���9b�#��L��ނN`�l�bʙ�ݛ7Ǣz�t��m%�����5��F�ɫ�"�)�r�f�%���N���t_z�9�ˈ|�ƥ|��!��Yw �r�����;}"��G��o���,p�kQy��n>f�?��(ox��(����[�Q��	t�1�~��
2��N,��Ә�����m����
[���%�=�����H��������t3?�����N^]Ȗ��p���c\L4�H�/�rw��͈'9\X�0��tJ���(E�3.�j��Y���/���BK����q����Q�2v8�4�C�3(b(f.-��������
��N%�q�z��h��:#�{7.^�C��0B��l�(���$��g�T�g��ײ�LLrg:�/�	�s$�X�y�������j�AD�N�:����X$D�N�!
���V
<�L�ZI��E�aQ.�h�ɡ;O�����
T��'f�#'��_�������WϹ�DܻyQ�҂E��7��J��
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    An unexpected error has occurred.
  Please try reproducing the error using
  the latest s3cmd code from the git master
  branch found at:
    https://github.com/s3tools/s3cmd
  and have a look at the known issues list:
    https://github.com/s3tools/s3cmd/wiki/Common-known-issues-and-their-solutions
  If the error persists, please report the
  following lines (removing any private
  info as necessary) to:
   s3tools-bugs@lists.sourceforge.net
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Invoked as: s3cmd -c .travis.s3cfg mv s3://baseautos3cmd-autotest-1/urandom1.bin s3://baseautos3cmd-autotest-1/urandom2.bin s3://baseautos3cmd-autotest-1/dir/
Problem: ParseError: not well-formed (invalid token): line 1, column 0
S3cmd:   2.0.1+
python:   2.7.14 (default, Nov 28 2017, 18:46:37) 
[GCC 4.8.4]
environment LANG=en_US.UTF-8
Traceback (most recent call last):
  File "s3cmd", line 3092, in <module>
    rc = main()
  File "s3cmd", line 3001, in main
    rc = cmd_func(args)
  File "s3cmd", line 902, in cmd_mv
    return subcmd_cp_mv(args, s3.object_move, "move", u"move: '%(src)s' -> '%(dst)s'")
  File "s3cmd", line 878, in subcmd_cp_mv
    response = process_fce(src_uri, dst_uri, extra_headers)
  File "/home/travis/build/s3tools/s3cmd/S3/S3.py", line 920, in object_move
    response_copy = self.object_copy(src_uri, dst_uri, extra_headers)
  File "/home/travis/build/s3tools/s3cmd/S3/S3.py", line 811, in object_copy
    acl = self.get_acl(src_uri)
  File "/home/travis/build/s3tools/s3cmd/S3/S3.py", line 943, in get_acl
    acl = ACL(response['data'])
  File "/home/travis/build/s3tools/s3cmd/S3/ACL.py", line 87, in __init__
    tree = getTreeFromXml(encode_to_s3(xml))
  File "/home/travis/build/s3tools/s3cmd/S3/Utils.py", line 106, in getTreeFromXml
    tree = ET.fromstring(xml)
  File "/opt/python/2.7.14/lib/python2.7/xml/etree/ElementTree.py", line 1311, in XML
    parser.feed(text)
  File "/opt/python/2.7.14/lib/python2.7/xml/etree/ElementTree.py", line 1659, in feed
    self._raiseerror(v)
  File "/opt/python/2.7.14/lib/python2.7/xml/etree/ElementTree.py", line 1523, in _raiseerror
    raise err
ParseError: not well-formed (invalid token): line 1, column 0
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    An unexpected error has occurred.
  Please try reproducing the error using
  the latest s3cmd code from the git master
  branch found at:
    https://github.com/s3tools/s3cmd
  and have a look at the known issues list:
    https://github.com/s3tools/s3cmd/wiki/Common-known-issues-and-their-solutions
  If the error persists, please report the
  above lines (removing any private
  info as necessary) to:
   s3tools-bugs@lists.sourceforge.net
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
----
The command "python ./run-tests-minio.py -c .travis.s3cfg -p baseauto" exited with 1.
cache.2
store build cache
0.01s
4.10schanges detected, packing new archive
uploading archive
after_script
0.06s$ killall minio
```


=== Log of my computer with debug output ===
```
$ ~/dev/s3cmd/ $ python s3cmd -c s3cfg_localminio mv s3://floautotests3cmd-autotest-1/urandom1.bin s3://floautotests3cmd-autotest-1/urandom2.bin s3://floautotests3cmd-autotest-1/dir/ -d
DEBUG: s3cmd version 2.0.1+
DEBUG: ConfigParser: Reading file 's3cfg_localminio'
DEBUG: ConfigParser: add_encoding_exts->
DEBUG: ConfigParser: add_headers->
DEBUG: ConfigParser: cache_file->
DEBUG: ConfigParser: cloudfront_host->cloudfront.amazonaws.com
DEBUG: ConfigParser: default_mime_type->binary/octet-stream
DEBUG: ConfigParser: delay_updates->False
DEBUG: ConfigParser: delete_after->False
DEBUG: ConfigParser: delete_after_fetch->False
DEBUG: ConfigParser: delete_removed->False
DEBUG: ConfigParser: dry_run->False
DEBUG: ConfigParser: encoding->UTF-8
DEBUG: ConfigParser: encrypt->False
DEBUG: ConfigParser: follow_symlinks->False
DEBUG: ConfigParser: force->False
DEBUG: ConfigParser: get_continue->False
DEBUG: ConfigParser: gpg_command->/usr/bin/gpg
DEBUG: ConfigParser: gpg_decrypt->%(gpg_command)s -d --verbose --no-use-agent --batch --yes --passphrase-fd %(passphrase_fd)s -o %(output_file)s %(input_file)s
DEBUG: ConfigParser: gpg_encrypt->%(gpg_command)s -c --verbose --no-use-agent --batch --yes --passphrase-fd %(passphrase_fd)s -o %(output_file)s %(input_file)s
DEBUG: ConfigParser: gpg_passphrase->pa...7_chars...e
DEBUG: ConfigParser: guess_mime_type->True
DEBUG: ConfigParser: human_readable_sizes->False
DEBUG: ConfigParser: invalidate_default_index_on_cf->False
DEBUG: ConfigParser: invalidate_default_index_root_on_cf->True
DEBUG: ConfigParser: invalidate_on_cf->False
DEBUG: ConfigParser: list_md5->False
DEBUG: ConfigParser: log_target_prefix->
DEBUG: ConfigParser: mime_type->
DEBUG: ConfigParser: multipart_chunk_size_mb->15
DEBUG: ConfigParser: preserve_attrs->True
DEBUG: ConfigParser: progress_meter->False
DEBUG: ConfigParser: proxy_host->
DEBUG: ConfigParser: proxy_port->0
DEBUG: ConfigParser: recursive->False
DEBUG: ConfigParser: recv_chunk->4096
DEBUG: ConfigParser: reduced_redundancy->False
DEBUG: ConfigParser: send_chunk->4096
DEBUG: ConfigParser: simpledb_host->sdb.amazonaws.com
DEBUG: ConfigParser: skip_existing->False
DEBUG: ConfigParser: urlencoding_mode->normal
DEBUG: ConfigParser: verbosity->WARNING
DEBUG: ConfigParser: website_error->
DEBUG: ConfigParser: website_index->index.html
DEBUG: ConfigParser: access_key->Q3...17_chars...F
DEBUG: ConfigParser: secret_key->zu...37_chars...G
DEBUG: ConfigParser: socket_timeout->500000
DEBUG: ConfigParser: enable_multipart->True
DEBUG: ConfigParser: host_base->127.0.0.1:9000
DEBUG: ConfigParser: host_bucket->127.0.0.1:9000
DEBUG: ConfigParser: use_https->False
DEBUG: ConfigParser: signature_v2->False
DEBUG: ConfigParser: bucket_location->us-east-1
DEBUG: Updating Config.Config cache_file -> 
DEBUG: Updating Config.Config follow_symlinks -> False
DEBUG: Updating Config.Config verbosity -> 10
DEBUG: Unicodising 'mv' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/urandom1.bin' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/urandom2.bin' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/dir/' using UTF-8
DEBUG: Command: mv
DEBUG: DeUnicodising u'urandom1.bin' using UTF-8
DEBUG: Unicodising 'urandom1.bin' using UTF-8
DEBUG: DeUnicodising u'urandom2.bin' using UTF-8
DEBUG: Unicodising 'urandom2.bin' using UTF-8
DEBUG: Applying --exclude/--include
DEBUG: CHECK: urandom1.bin
DEBUG: PASS: u'urandom1.bin'
DEBUG: CHECK: urandom2.bin
DEBUG: PASS: u'urandom2.bin'
INFO: Summary: 2 remote files to move
DEBUG: CreateRequest: resource[uri]=/urandom1.bin
DEBUG: Using signature v4
DEBUG: get_hostname(floautotests3cmd-autotest-1): 127.0.0.1:9000
DEBUG: canonical_headers = host:127.0.0.1:9000
x-amz-content-sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
x-amz-date:20180603T154250Z

DEBUG: Canonical Request:
GET
/floautotests3cmd-autotest-1/urandom1.bin
acl=
host:127.0.0.1:9000
x-amz-content-sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
x-amz-date:20180603T154250Z

host;x-amz-content-sha256;x-amz-date
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
----------------------
DEBUG: signature-v4 headers: {'x-amz-content-sha256': u'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855', 'Authorization': u'AWS4-HMAC-SHA256 Credential=Q3AM3UQ867SPQQA43P2F/20180603/us-east-1/s3/aws4_request,SignedHeaders=host;x-amz-content-sha256;x-amz-date,Signature=29f924538751aff61b586a78fb0ff9de4039afb52ee4f84d70c5e9684d26214f', 'x-amz-date': '20180603T154250Z'}
DEBUG: Processing request, please wait...
DEBUG: get_hostname(floautotests3cmd-autotest-1): 127.0.0.1:9000
DEBUG: ConnMan.get(): creating new connection: http://127.0.0.1:9000
DEBUG: non-proxied HTTPConnection(127.0.0.1, 9000)
DEBUG: format_uri(): /floautotests3cmd-autotest-1/urandom1.bin?acl
DEBUG: Sending request method_string='GET', uri=u'/floautotests3cmd-autotest-1/urandom1.bin?acl', headers={'x-amz-content-sha256': u'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855', 'Authorization': u'AWS4-HMAC-SHA256 Credential=Q3AM3UQ867SPQQA43P2F/20180603/us-east-1/s3/aws4_request,SignedHeaders=host;x-amz-content-sha256;x-amz-date,Signature=29f924538751aff61b586a78fb0ff9de4039afb52ee4f84d70c5e9684d26214f', 'x-amz-date': '20180603T154250Z'}, body=(0 bytes)
DEBUG: ConnMan.put(): connection put back to pool (http://127.0.0.1:9000#1)
DEBUG: Response:
{'data': '\x1d\x16 zb\x9f\x1c\xb8n]\xae\xa2\xd2]3>4l\xf5\xd8\x01\xf2\xbb\xb3\x9e\xe9f\x12`G:G\xf5Cw\xd3O[F\x97\xcb\x1a\xb5i>=\xc6\x14~\xba\xc8\xa9\xbeL,D \x8d\xbf\xe9\xb6\x1f\xcd\xdc\x03\xa6\x95n\xb0v\xc8\x8a\xee\x15\x97\xbf\x93\xc6\nv\xb0C`\xa5\xd4\xe3\xe8\xee\xe8\x1e\x1fB>e/\x85\xc2!\xb6\xfa\x18/\x86\x9f\x85\xe9\x85;[\xa78\x9e/R\xc0\xcf\xeb\xf3\x98\xfe\xdb0`r\xb5\xab\x99\x10\xbc#A\x02\xae\xff\x96\xf9_}\xbc\xdd\x96\x95\x83\x1c\x7fJLOfs\x1b\xb1\x0c\xab}4g\xea\xdd\x07\xd03t\x8c\x8e\xba\xe8o\x8cyp\xa4\x8f\x89\xff\x8b\xfc\xba\x07.\x0b\x98|\xb2K9_LD\x80\x15\xd6\xee\x8c\xaf\xeah\x8c|s\xd3D\xfc\xd6\xc3\xdc\xc5\x1a\xd0A\x8b\xf76\x0f\x85\x8b@.]\x89\xb1\xe6\x93BLC\xedP@\x959\xf7\x0f\x89\x15\x15%o\xbc\xf3\x8e\xe4F\xac6\xa16\xb7\x0fG\xdd\xe1\xeen\xd3E\xd8\xbf\x87qK-u\\\x84? G\xc2\x1c0\x1f\x99L\xa8L\xab\xda\xb2\xc1QZ\x80\x10Z\xb1\x8aW\x95}\xa4g\xd2%&{Kz\xaa\x8a$\xa1\xb3\xb7\xeci\xf8\x82\x11T&\xf3\x00\xa1\x96\xffs\xff\xeb\x8b\x05\x9cy+vU\r\xf6~\xb6{\xc57V\xfd\xc3\x95H;\xb0\x9e\xea\xdc/n\xc4A\xbc\x89\xfc\x03t\xf49\x84\x99amWN(\xe6\xb8\xfa\xb5\xb0IA\xd5\xf2m\xc3\x02\x1cn\x95b\x1c\xa2\xc6?[fH\xcc~\x9cK\x99\xe9\xa88JT\\\x8c\x93S\x18)w\xe4\xf8\x18\xce$9\xa5\x97\x9d<be?\xc1\xf5\xef+\xa9\x88\x00 2\xd65\x1a\xa2\xc6%\x11\xe2#\x8c\x7f\x97\xc2[H]\x08\xe8\x12-\xa6\xd5\x1bH\xef\xd2\x88Cm\xae\xc4\xf2\xf0C|\x7f\x95&\x88\x0f\x9a\xba>\x8foR\x91\xaa=\tn/\x9b\xe6\x01\x02\xf0\xa9\xcd\x01IQ&\x9e\x1c\xc6:\xd4\xbdOD4c~B\xcaC\xee\xbbPg\xcdM\xf2\xec\x05\xbd\x02\x8e\xdd\x8b!=4SO\xe9b%?\xeap\x1a>k<iL\xba\xc0U.\xa0nt\x92\x8d\xc5\x06\xe6\xba]\xe9\xdc\xc07\xcb\xf25\xcam\xbb\x8d\xb6\xc5\xb1\x83\xed\xef\xe2\x0b\x14\xb8p\xebnl\xd7\x8f\xa0+\x9a\xfcJP\xc0\xe6\xd4\xda4|h?\x97V\x7f\xb4\x17\xc0\x0c\xa2\xc8\x9e\xe9\xd4\x9b\x11\xd6\xeb\xf3\x90\xdbd\x10\xd7\x94\'\x9e7&\x9d|\xeazj\xb1f\xc0\x00\x11\xf2s\xfchr\x0fd\x0b5\x90\r\x98\x8e\xf3\x97\x04\xb1\x96\xd7\xd5W\x13\xa76\x1d\np\xa9\xd2\xbb\xd3\xed\xac\x1a9\xe3T \xff\xfe\'\xd3\xe4\xca\xbf"\x9b\xf9}\x16\xf0\x9b\xafe\x0e\xd9\xfe\xdd\x1b\xc2\x95\xa5,\xd1\xe9A,\x82#\x1fD\xab\xd3\xec\x15*\xd2\xbdJ\xf5\xe6z\xb64\x10q\x82<\x17E\xbaT\xf5\xd3\x16\xe7\r)XN\xfb.\x044\xd6\x88\x1f\xad\\\xa7\x8bf! \xc5:\x95\x02\x12\x8d_\xe0\xaf+\xba\xbex\x868g\x1c\x15\x98d\x19\xb8\x11\x01\xce\xf2\x9d\x0e\x98.\xe4X`$Z\x8b7\xe5\xee\xe2Dtz\xdd\xe3\xa0\x10p\xf9\xc2\x9cF\x0eG\x89M\xff\xeb\x95\r\x1e\t\xa0\x065\xa3Le\xf0%\x03\xd8\xba\xe3\xbc]\xf5k\x82\xe8E\x83701L\xc3\xf1\xcb\xef\x92g \\\x1c\xff\x08\xe4F\xba\x96g(\x98\xd2\xa0P\xc9\n\xda\xe7{\x0e"v$\xf5,\xfb \xd1z\xa5\xed\xac\xce\t\tBL$\xad\xb5OA\xc55<KK\xd9p|\xde\xe0\xf4\x14\xf0c\xbeq\x96Ax\xba\xe8iU\xd6\xbb\x8f\xfcP\x1b\x8fc\xe0-^\xe9\x87\r\x12\x9e\xa6\x0f+\xae\x18P+I \x18\xc0jr\x02\xfa\x89&\x82\xb8\x0c\x86\x9f\xbd\r\xd4\xa9\xd4:\x90n8\x99\xfc,\xa7b\xaaok(nn~\xde\xc8\xc8|\x92K\x8a\xc3\xebY\x89\x88\xec\x9c\x8fd\x9f\xe72\xf2g\x1c\x92\xd2^\xbb\x7f\xbb@MH\xeb\x00%p\xd7\xb6=O\xdd\x07\xb6y\xbc\xf6\xfb\x9c\x1c\xb2\xe8\x1eAI\r_.E\x11\xf7\x97\xa6\xc8\xe2"/\'U}5\x1b',
 'headers': {'accept-ranges': 'bytes',
             'content-length': '1024',
             'content-security-policy': 'block-all-mixed-content',
             'content-type': 'application/octet-stream',
             'date': 'Sun, 03 Jun 2018 15:42:50 GMT',
             'etag': '"59ff0ca07a1ba28865a5009bfda7541f"',
             'last-modified': 'Sun, 03 Jun 2018 15:41:42 GMT',
             'server': 'Minio/RELEASE.2018-05-25T19-49-13Z (linux; amd64)',
             'vary': 'Origin',
             'x-amz-bucket-region': 'us-east-1',
             'x-amz-meta-s3cmd-attrs': 'atime:1528040502/ctime:1528040502/gid:1000/gname:flo/md5:59ff0ca07a1ba28865a5009bfda7541f/mode:33188/mtime:1528040502/uid:1000/uname:flo',
             'x-amz-request-id': '1534B0CD93CBCC2F',
             'x-xss-protection': '"1; mode=block"'},
 'reason': 'OK',
 's3cmd-attrs': {'atime': '1528040502',
                 'ctime': '1528040502',
                 'gid': '1000',
                 'gname': 'flo',
                 'md5': '59ff0ca07a1ba28865a5009bfda7541f',
                 'mode': '33188',
                 'mtime': '1528040502',
                 'uid': '1000',
                 'uname': 'flo'},
 'status': 200}
ERROR: Error parsing xml: not well-formed (invalid token): line 1, column 0
ERROR:  zb��n]���]3>4l��򻳞�f`G:G�Cw�O[F���i>=�~�ȩ�L,D ��������n�vȊ�����
v�C`������B>e/��!��/�����;[�8�/R�������0`r����#A����_}�ݖ��JLOfs�
                                                                       �}4g���3t����o�yp�������.
�~�{�7V�ÕH;����/n�A���t�9��amWN(�����IA��m�n�b��?[fH�~�K���8JT\��S)w���$9���<be?���+�� 2�5��%�#���[H�-���҈Cm����C|�&���>�oR��=�Bn/�����IQ&��:ԽOD4c~B�C��Pg�M����݋!=4SO�b%?�p>k<iL��U.�nt�����]���7��5�m���ű�����s����y+vU
                                                                                                                                                                                                                              �p�nl׏�+��JP����4|h?�V��
          �Ȟ�ԛ�����dה'�7&�|�zj�f��s�hrd
��������W�6                            5�
XN�.4ֈ�5�Le�%غ��]�k��E�701L�����g \�F��g(�ҠP��7���Dtz���p�FG�M���
��+�P+I �jr��&��		BL$��OA�5<KK�p|����c�q�Ax��iUֻ��P�c�-^��
_.E�����"/'U}5n~���|�K���Y��윏d��2�g��^��@MH�%p׶=O��y������AI

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    An unexpected error has occurred.
  Please try reproducing the error using
  the latest s3cmd code from the git master
  branch found at:
    https://github.com/s3tools/s3cmd
  and have a look at the known issues list:
    https://github.com/s3tools/s3cmd/wiki/Common-known-issues-and-their-solutions
  If the error persists, please report the
  following lines (removing any private
  info as necessary) to:
   s3tools-bugs@lists.sourceforge.net


!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

DEBUG: Unicodising 's3cmd' using UTF-8
DEBUG: Unicodising '-c' using UTF-8
DEBUG: Unicodising 's3cfg_localminio' using UTF-8
DEBUG: Unicodising 'mv' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/urandom1.bin' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/urandom2.bin' using UTF-8
DEBUG: Unicodising 's3://floautotests3cmd-autotest-1/dir/' using UTF-8
DEBUG: Unicodising '-d' using UTF-8
Invoked as: s3cmd -c s3cfg_localminio mv s3://floautotests3cmd-autotest-1/urandom1.bin s3://floautotests3cmd-autotest-1/urandom2.bin s3://floautotests3cmd-autotest-1/dir/ -d
Problem: ParseError: not well-formed (invalid token): line 1, column 0
S3cmd:   2.0.1+
python:   2.7.12 (default, Dec  4 2017, 14:50:18) 
[GCC 5.4.0 20160609]
environment LANG=fr_FR.UTF-8

Traceback (most recent call last):
  File "s3cmd", line 3092, in <module>
    rc = main()
  File "s3cmd", line 3001, in main
    rc = cmd_func(args)
  File "s3cmd", line 902, in cmd_mv
    return subcmd_cp_mv(args, s3.object_move, "move", u"move: '%(src)s' -> '%(dst)s'")
  File "s3cmd", line 878, in subcmd_cp_mv
    response = process_fce(src_uri, dst_uri, extra_headers)
  File "/home/flo/dev/s3cmd/S3/S3.py", line 920, in object_move
    response_copy = self.object_copy(src_uri, dst_uri, extra_headers)
  File "/home/flo/dev/s3cmd/S3/S3.py", line 811, in object_copy
    acl = self.get_acl(src_uri)
  File "/home/flo/dev/s3cmd/S3/S3.py", line 943, in get_acl
    acl = ACL(response['data'])
  File "/home/flo/dev/s3cmd/S3/ACL.py", line 87, in __init__
    tree = getTreeFromXml(encode_to_s3(xml))
  File "/home/flo/dev/s3cmdS3/Utils.py", line 106, in getTreeFromXml
    tree = ET.fromstring(xml)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1311, in XML
    parser.feed(text)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1653, in feed
    self._raiseerror(v)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1517, in _raiseerror
    raise err
ParseError: not well-formed (invalid token): line 1, column 0

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
    An unexpected error has occurred.
  Please try reproducing the error using
  the latest s3cmd code from the git master
  branch found at:
    https://github.com/s3tools/s3cmd
  and have a look at the known issues list:
    https://github.com/s3tools/s3cmd/wiki/Common-known-issues-and-their-solutions
  If the error persists, please report the
  above lines (removing any private
  info as necessary) to:
   s3tools-bugs@lists.sourceforge.net
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```



## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used (`minio version`): Version: 2018-05-25T19:49:13Z
* Operating System and version (`uname -a`): Linux

**Comment by harshavardhana** [comment]
This is already fixed #5999 , will be available in the next release.

**Comment by fviard** [comment]
Awesome, thank you.
I did look into existing issues but didn't check the recent commits.
